src/iframe/src/Iframe.js

```diff
@@ -173,6 +174,10 @@ class Iframe extends Component {
       run: true,
       sound: true
     }
+
+    this._pixelBuffer = new ArrayBuffer(4 * CANVAS_SIZE * CANVAS_SIZE)
```

Comment:
For my sake more than anything else, could you add a comment on each of these three lines, explaining what they do? It will make it easier for me to understand the rest of the new shiny renderer.

Reply:
Done.